src/debug.h

```diff
@@ -38,7 +38,7 @@
 	false  // monitor accel for (triple) tap events and send them. Uses more cpu,
 		   // disable if problems. Server does nothing with value so disabled atm
 #define SEND_ACCELERATION true  // send linear acceleration to the server
-
+#define USE_NONBLOCKING_CALIBRATION
```

Comment:
```suggestion
#define USE_NONBLOCKING_CALIBRATION true
```
Let's check defines to `true`, so we could set them to `false` too during compilation, for example.